src/main/java/com/google/devtools/build/lib/remote/RemoteSpawnRunner.java

```diff
@@ -322,9 +321,20 @@ public SpawnResult exec(Spawn spawn, SpawnExecutionContext context)
                 spawnMetrics.setUploadTime(
                     uploadTime.elapsed().minus(networkTime.getDuration().minus(networkTimeStart)));
               }
+              final AtomicBoolean reportedExecuting =
+                  new AtomicBoolean(false);
               ExecuteResponse reply;
               try (SilentCloseable c = prof.profile(REMOTE_EXECUTION, "execute remotely")) {
-                reply = remoteExecutor.executeRemotely(request);
+                reply = remoteExecutor.executeRemotely(request, (operation) -> {
```

Comment:
My personal preference would be to not use a closure here but an anonymous inner class and move the AtomicBoolean as a field into the class. Feel free to ignore this.